luadox/main.py

```diff
@@ -186,6 +192,12 @@ def main():
                    help='Custom CSS file (html renderer)')
     p.add_argument('--favicon', action='store', type=str, metavar='FILE',
                    help='Path to favicon file (html renderer)')
+    p.add_argument('--head_template', action='store', type=str, metavar='FILE',
```

Comment:
Don't use underscores for command line flags, use dashes instead.  So `--head-template`, `--foot-template`, and `--search-template`.